MobiFlight/ExecutionManager.cs

```diff
@@ -1005,6 +1008,7 @@ void timer_Stopped(object sender, EventArgs e)
 #endif
             inputCache.Clear();
             inputActionExecutionCache.Clear();
+            mobiFlightCache.ActivateConnectedModulePowerSave();

```

Comment:
Was there a specific reason that you had decided to take the timer event?

Better options are `MobiFlightModule.Stop()` method, `MobiFlightCache.Stop()` method for it, or `ExecutionManager.Stop()` method.

I am not even sure why are really doing this. What's the sense in it? On Stop, we don't care, and the module should take care which it would even without this message.

Reply:
> What's the sense in it? On Stop, we don't care, and the module should take care which it would even without this message.

It's not strictly necessary, but it does make the end user experience pretty nice. In the normal operating case, all your connected modules will wake up as soon as run happens and shut down as soon as MobiFlight stops.

Reply:
Funny but true story: I just finished doing more testing with the old code and closed Mobiflight. My radio stack is still has all eight frequencies lit up and I went "wtf, why didn't that turn off?". Oh, right, because with the old code I have to wait 15 minutes for the board to time itself out. LAME!!! 😀

Reply:
I literally just checked out this branch so I could hit the stop button and turn the boards off 🤣

Reply:
Same here. The other statements are simply wrong: 

<img width="654" alt="image" src="https://github.com/MobiFlight/MobiFlight-Connector/assets/86157512/bc664b04-6b63-4410-9eb9-af247d6b42f8">

They should be moved out to the `ExceutionManager.Stop()`

`timer.enabled=false` is only called in two places:
In `ExecutionManager.Stop()`  or when an exception is raised in the timer_tick - this should be changed to `ExecutionManager.Stop()` to make sure everything is stopped correctly.